loader/loader.c

```diff
@@ -2335,7 +2309,7 @@ static VkResult loader_read_layer_json(const struct loader_instance *inst, struc
         GET_JSON_ITEM(inst, functions, vkGetDeviceProcAddr)
         if (vkGetInstanceProcAddr != NULL) {
             strncpy(props->functions.str_gipa, vkGetInstanceProcAddr, sizeof(props->functions.str_gipa));
-            if (version.major > 1 || version.minor >= 1) {
+            if (version.major >= 1 && version.minor >= 1) {
```

Comment:
Same here, it will break if 2.0 because minor is < 1

Reply:
Good catch. Writing version matching is hard, and took me a long time to even think this was valid.